server/cluster/unsafe_recovery_controller.go

```diff
@@ -43,51 +43,51 @@ const (
-//  +-----------+       +-----------+     |     |           |     |
-//                                        |     +-----------+     |
-//                                        |           |           |
-//                                        |           |           |
-//                                        |           v           |
-//                                        |     +-----------+     |
-//                                        |     |           |     |
-//                                        |     |   force   |     |
-//                                        |     | LeaderFor |-----+
-//                                        |     |CommitMerge|     |
-//                                        |     |           |     |
-//                                        |     +-----------+     |
-//                                        |           |           |
-//                                        |           |           |
-//                                        |           v           |
-//                                        |     +-----------+     |     +-----------+
-//                                        |     |           |     |     |           |        +-----------+
-//                                        |     |  force    |     |     | exitForce |        |           |
-//                                        |     |  Leader   |-----+---->|  Leader   |------->|  failed   |
-//                                        |     |           |     |     |           |        |           |
-//                                        |     +-----------+     |     +-----------+        +-----------+
-//                                        |           |           |
-//                                        |           |           |
-//                                        |           v           |
-//                                        |     +-----------+     |
-//                                        |     |           |     |
-//                                        |     |  demote   |     |
-//                                        +-----|  Voter    |-----|
-//                                              |           |     |
-//                                              +-----------+     |
-//                                                    |           |
-//                                                    |           |
-//                                                    v           |
-//                      +-----------+           +-----------+     |
-//  +-----------+       |           |           |           |     |
-//  |           |       | exitForce |           |  create   |     |
-//  | finished  |<------|  Leader   |<----------|  Region   |-----+
-//  |           |       |           |           |           |
-//  +-----------+       +-----------+           +-----------+
 //
+//	                    +-----------+           +-----------+
```

Comment:
why change

Reply:
This is auto formatted by `go fmt` I think